### ISSUE TITLE
[highlighting] Add screenreader announcement

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#3825]
- [x] The screen reader should announce sth in the content of 'Important: Search term is emphasized' or "Important: You searched for". In screen reader avoid the term 'highlighting'. 
- [x] I can successfully use this feature with Voice Over in Safari. 
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

**Comment by christinach** [comment]
resolved by https://github.com/pulibrary/orangelight/pull/3832/files
